build/vm-monitor/Dockerfile

```diff
@@ -8,14 +8,14 @@ RUN apk add musl-dev git
 #
 # Note: unlike in the stackoverflow post we access the branches endpoint so that
 # `latest_commit` will change upon a commit in any branch
-ADD "https://api.github.com/repos/neondatabase/vm-monitor/branches" latest_commit
+ADD "https://api.github.com/repos/neondatabase/neon/branches" latest_commit
```

Comment:
H, probably we should only care about the branch we're building from - otherwise the rate of new commits on _any_ branch in `neondatabase/neon` will constantly invalidate the cache :sweat_smile: